packages/fluentui/accessibility/src/behaviors/Tree/treeItemBehavior.ts

```diff
@@ -22,64 +22,73 @@ import treeTitleBehavior from './treeTitleBehavior';
+const treeItemBehavior: Accessibility<TreeItemBehaviorProps> = props => {
+  const definition = {
+    attributes: {
+      root: {
+        role: 'none',
+        ...(props.hasSubtree && {
+          'aria-expanded': props.expanded,
+          'aria-selected': props.selectable ? props.selected || false : undefined,
+          tabIndex: -1,
+          [IS_FOCUSABLE_ATTRIBUTE]: true,
+          role: 'treeitem' as AriaRole,
+          'aria-setsize': props.treeSize,
+          'aria-posinset': props.index,
+          'aria-level': props.level,
+        }),
       },
-      ...(isSubtreeExpanded(props) && {
-        collapse: {
-          keyCombinations: [{ keyCode: keyboardKey.ArrowLeft }],
-        },
-        focusFirstChild: {
-          keyCombinations: [{ keyCode: keyboardKey.ArrowRight }],
-        },
-        focusParent: {
-          keyCombinations: [{ keyCode: keyboardKey.ArrowLeft }],
+    },
+    keyActions: {
+      root: {
+        performClick: {
+          keyCombinations: [{ keyCode: keyboardKey.Enter }, { keyCode: keyboardKey.Spacebar }],
         },
-      }),
-      ...(!isSubtreeExpanded(props) &&
-        props.hasSubtree && {
-          expand: {
+        ...(isSubtreeExpanded(props) && {
+          collapse: {
+            keyCombinations: [{ keyCode: keyboardKey.ArrowLeft }],
+          },
+          focusFirstChild: {
             keyCombinations: [{ keyCode: keyboardKey.ArrowRight }],
           },
           focusParent: {
             keyCombinations: [{ keyCode: keyboardKey.ArrowLeft }],
           },
         }),
-      expandSiblings: {
-        keyCombinations: [{ keyCode: keyboardKey['*'] }],
-      },
-      ...(props.selectable && {
-        performClick: {
-          keyCombinations: props.hasSubtree ? [{ keyCode: keyboardKey.Enter }] : [{ keyCode: keyboardKey.Spacebar }],
+        ...(!isSubtreeExpanded(props) &&
+          props.hasSubtree && {
+            expand: {
+              keyCombinations: [{ keyCode: keyboardKey.ArrowRight }],
+            },
+            focusParent: {
+              keyCombinations: [{ keyCode: keyboardKey.ArrowLeft }],
+            },
+          }),
+        expandSiblings: {
+          keyCombinations: [{ keyCode: keyboardKey['*'] }],
         },
-        performSelection: {
-          keyCombinations: [{ keyCode: keyboardKey.Spacebar }],
-        },
-      }),
+        ...(props.selectable && {
+          performClick: {
+            keyCombinations: props.hasSubtree ? [{ keyCode: keyboardKey.Enter }] : [{ keyCode: keyboardKey.Spacebar }],
+          },
+          performSelection: {
+            keyCombinations: [{ keyCode: keyboardKey.Spacebar }],
+          },
+        }),
+      },
     },
-  },
-  childBehaviors: {
-    title: treeTitleBehavior,
-  },
-});
+    childBehaviors: {
+      title: treeTitleBehavior,
+    },
+  };
+
+  if (process.env.NODE_ENV !== 'production' && !props.hasSubtree) {
```

Comment:
To avoid these conditions I suggest in another task that we refactor `Tree` and get ride of `TreeTitle` and have only `TreeItem` inside the tree. What do you think @kolaps33 @layershifter @jurokapsiar